.github/workflows/claude-assistant.yml

```diff
@@ -0,0 +1,46 @@
+name: Claude Assistant
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  post_claude_response:
+    if: contains(github.event.comment.body, '@!claude')
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Get stripped comment body
+      id: get-comment-body
+      run: echo comment=$(echo ${{github.event.comment.body}} | sed 's/@!claude//g') >> $GITHUB_OUTPUT
+
+    - name: Get Claude's response to issue comment
+      id: claude-response
+      if: steps.get-comment-body.outputs.comment != ''
+      env:
+        ANTHROPIC_API_KEY: ${{ secrets.ANTHROPIC_API_KEY }}
+      run: |
+        echo response=$(curl https://api.anthropic.com/v1/messages \
+            --header "x-api-key: $ANTHROPIC_API_KEY" \
+            --header "anthropic-version: 2023-06-01" \
+            --header "content-type: application/json" \
+            --data \
+            '{
+                "model": "claude-3-haiku-20240307",
+                "max_tokens": 1024,
+                "messages": [
+                    {"role": "user", "content": "${{ steps.get-comment-body.outputs.comment }}"}
```

Comment:
Happy to merge at this, but do a follow up PR with all the issue history included